Checks/PWR002/README.md

```diff
@@ -90,10 +90,72 @@ void example() {
 
 #### Fortran
 
->**Note**  
->The Fortran programming language does not allow to declare variables inside
->loops, in the statements of the loop body. In modern Fortran there are some
->circumstances where this is allowed, such as `do concurrent` statements.
+Fortran 2008 introduced the `BLOCK` construct. This feature is rather
+convenient to create sections of code for enhanced readability in larger
+programs. Conveniently, it supports new variable declarations.
```

Comment:
```suggestion
Fortran 2008 introduced the `BLOCK` construct. This feature allows to organize
code within larger programs by grouping sections together. Conveniently, the
`BLOCK` construct supports declaring variables within those sections, further pro
```

What do you think? This avoids repeating "convenient" in the same paragraph.

Reply:
Done! :)